inc/inventoryaction.class.php

```diff
@@ -179,7 +180,7 @@ public static function postItemForm($item)
 
                 echo $out;
 
-                $url = Plugin::getWebDir('databaseinventory') . '/ajax/agent.php';
+                $url = plugin_databaseinventory_geturl() . 'ajax/agent.php';
```

Comment:
Why not a function, but it's only called once, I think, so it's not necessary.
```suggestion
                $url = $CFG_GLPI['url_base'] . '/plugins/databaseinventory/ajax/agent.php';
```